src/Datadog.Trace/Util/Http/HttpRequestExtensions.Core.cs

```diff
@@ -24,7 +25,7 @@ internal static partial class HttpRequestExtensions
             {
                 if (!k.Equals("cookie", System.StringComparison.OrdinalIgnoreCase))
                 {
-                    headersDic.Add(k, request.Headers[k]);
+                    headersDic.Add(k.ToLower(CultureInfo.InvariantCulture), request.Headers[k]);
```

Comment:
Any reason to not use `ToLowerInvariant()`?
```suggestion
                    headersDic.Add(k.ToLowerInvariant(), request.Headers[k]);
```

Reply:
have to admit the main reason I went with ToLower is that it's higher in the autocomplete list :)

Reply:
fixed in a2e44b78